VSRAD.Syntax/FunctionList/FunctionListControl.xaml.cs

```diff
@@ -10,7 +10,6 @@
 using Task = System.Threading.Tasks.Task;
 using System.Threading;
 using System.Linq;
-using VSRAD.Syntax.Options;
 
 namespace VSRAD.Syntax.FunctionList
```

Comment:
In this file all private properties will be better to rename so they will start with `_`.

(for example, `private SortState SortState` is little bit confusing)

Reply:
This is a good comment, but not within the scope of this pull request.